server/test/mapping/mapping.test.js

```diff
@@ -0,0 +1,14 @@
+const app = require('../../app');
+const supertest = require('supertest');
+const request = supertest(app);
+
+// Mapping Config
+describe('GET /api/specification/mapping', () => {
```

Comment:
You have a chance to write english, why are you saying GET ... what is the intent of this test? what are you checking for? Sometimes non technical folks understand a codebase by reading the test names. @pcgamer1

Reply:
Improved the description of the test below the describe statement, in this describe statement it just means that we are covering all the tests under this endpoint.